PluginGemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-
+ruby '2.7.4'
```

Comment:
I still think this may cause conflicts when using the plugin due to conflicting ruby version requirements

Reply:
```
Your Gemfile lists the gem rubocop (~> 1.12.0) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once (per group).
While it's not a problem now, it could cause errors if you change the version of one of them later.
Your Gemfile lists the gem rubocop-performance (~> 1.10.1) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once (per group).
While it's not a problem now, it could cause errors if you change the version of one of them later.
Your Gemfile lists the gem redmine_crm (>= 0) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once (per group).
While it's not a problem now, it could cause errors if you change the version of one of them later.
Your Ruby version is 2.6.5, but your Gemfile specified 2.7.4
Your Gemfile lists the gem rubocop (~> 1.12.0) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once (per group).
While it's not a problem now, it could cause errors if you change the version of one of them later.
Your Gemfile lists the gem rubocop-performance (~> 1.10.1) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once (per group).
While it's not a problem now, it could cause errors if you change the version of one of them later.
Your Gemfile lists the gem redmine_crm (>= 0) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once (per group).
While it's not a problem now, it could cause errors if you change the version of one of them later.
Your Ruby version is 2.6.5, but your Gemfile specified 2.7.4
```

Reply:
I think this issue comes from renuo-redmine and redmine_tracky having different/conflicting versions. Fixing that probably will take a bit longer than just this small PR...

Reply:
It is fixed by just *not* merging your PR :wink:

Reply:
Without changing the versions, I cant even run `rails c` locally:
![image](https://user-images.githubusercontent.com/75726773/219342283-55bcb687-e628-4640-ba3f-7341702e1d74.png)

Reply:
And were upgrading redmine to ruby 2.7.7:
https://github.com/renuo/renuo-redmine/pull/35